omnibridge/wrappers/wrapper_interfaces/file_generating_model_wrapper.py

```diff
@@ -0,0 +1,8 @@
+from abc import abstractmethod, ABC
+from typing import Any
+
+
+class FileGenModelWrapper(ABC):
+    @abstractmethod
+    def prompt_and_generate_files(self, prompt: str) -> Any:
```

Comment:
Given its a file generating wrapper, can we force better typing than Any? for example list of paths